modules/contact-form/grunion-contact-form.php

```diff
@@ -339,6 +339,16 @@ private static function register_contact_form_blocks() {
 	}
 
 	public static function gutenblock_render_form( $atts, $content ) {
+		// Render fallback in other contexts than frontend (i.e. feed, emails, API, etc.).
+		if ( ! jetpack_is_frontend() ) {
+			return sprintf(
+				'<div class="%1$s"><a href="%2$s" target="_blank" rel="noopener noreferrer">%3$s</a></div>',
+				esc_attr( Blocks::classes( 'contact-form', $atts ) ),
+				esc_url( get_the_permalink() ),
+				esc_html__( 'Submit a contact form.', 'jetpack' )
```

Comment:
Maybe just "form", since folks can now use the block to build other types of forms?

```suggestion
				esc_html__( 'Submit a form.', 'jetpack' )
```